client/src/sass/application.scss

```diff
@@ -364,51 +363,16 @@ table {
     }
 
     &:not(.expanded) {
-      // TODO: $horizontal-margins might change to $expanded-horizontal-margins/3 + $menu-width/2
       @include overflow-datatable(breakpoint(lg), $not-expanded-horizontal-margins + $menu-width/2);
```

Comment:
@rigelk when you rebased you forgot to change the `$horizontal-margins` to `$expanded-horizontal-margins/3` here, no ?

Reply:
Ah, right. I assumed you had forgotten to remove the TODO since your PR wasn't a draft anymore.

Reply:
I'll change that in the next commit I do.

Reply:
Thanks !

Reply:
@kimsible I integrated the change in c2a89b70c